packages/manager/src/features/CloudPulse/shared/CloudPulseResourcesSelect.tsx

```diff
@@ -79,9 +81,22 @@ export const CloudPulseResourcesSelect = React.memo(
 
     return (
       <Autocomplete
-        onChange={(e, resourceSelections: CloudPulseResources[]) => {
+        onChange={(
+          _: React.SyntheticEvent,
+          resourceSelections: CloudPulseResources[]
```

Comment:
It's okay to just let typescript infer the types here

```suggestion
        onChange={(e, resourceSelections) => {
```